lib/plugins/inventory.js

```diff
@@ -122,17 +124,26 @@ function inject (bot, { hideErrors }) {
       })
     } else if (bot.supportFeature('useItemWithOwnPacket')) {
       bot._client.write('use_item', {
-        hand: offHand ? 1 : 0
+        hand: offHand ? 1 : 0,
+        sequence: ++sequence
```

Comment:
Does this always increment? Can you put this increment operation up top as opposed to inside the packet body?

Reply:
Yes, to my knowledge it always increments. Decided to spam until it passed 8bit mark; I'm not going to continue doing so further.

I'll update the code, but personally I feel moving it upwards is unnecessary.

![image](https://github.com/user-attachments/assets/03229a06-a164-47ef-bcb2-d3e23d36e799)